### PR TITLE
Added text for petting Holoparasites

### PR DESCRIPTION
**Pull request by MaxNox7**
I sure do hope this works...

**Changelog**
Added text for both succeeding and failing at petting a Holoparasite

### CODE REVIEW

File: Resources/Locale/en-US/interaction/interaction-popup-component.ftl [hunk 1]
```diff
@@ -27,6 +27,7 @@ petting-failure-crab = You reach out to pet {THE($target)}, but {SUBJECT($target
 petting-failure-goat = You reach out to pet {THE($target)}, but {SUBJECT($target)} stubbornly refuses!
 petting-failure-goose = You reach out to pet {THE($target)}, but {SUBJECT($target)} {CONJUGATE-BE($target)} too horrible!
 petting-failure-sloth = You reach out to pet {THE($target)}, but {SUBJECT($target)} somehow dodge with ludicrous speed!
+petting-failure-holo = You reach out to pet {THE($target)}, but it's spikes almost impale your hand!
```
Comment (Willhelm53) [inline Resources/Locale/en-US/interaction/interaction-popup-component.ftl:31]: Change "it's" to {POSS-ADJ($target)} and you should be good. Also maybe close this PR and make this change in your first branch

**Review by Willhelm53 — CHANGES_REQUESTED** [review]

**Comment by Peptide90** [comment]
Should probably be apart of #6727

**Comment by Zymem** [comment]
Just a warning, there's a typo in the text. 
"it's" should be "its".

**Comment by Willhelm53** [comment]
> Just a warning, there's a typo in the text. "it's" should be "its".

Changing that to {POSS-ADJ($target)} will take care of this for you. Good work though!

**Review by mirrorcult — CHANGES_REQUESTED** [review]
add the complementary component to the holopara

File: Resources/Locale/en-US/interaction/interaction-popup-component.ftl (hunk 1 above)
Comment (Willhelm53) [reply Resources/Locale/en-US/interaction/interaction-popup-component.ftl:31]: >Also maybe close this PR and make this change in your first branch

Ignore this part now.

File: Resources/Locale/en-US/interaction/interaction-popup-component.ftl [hunk 2]
```diff
@@ -27,6 +27,7 @@ petting-failure-crab = You reach out to pet {THE($target)}, but {SUBJECT($target
 petting-failure-goat = You reach out to pet {THE($target)}, but {SUBJECT($target)} stubbornly refuses!
 petting-failure-goose = You reach out to pet {THE($target)}, but {SUBJECT($target)} {CONJUGATE-BE($target)} too horrible!
 petting-failure-sloth = You reach out to pet {THE($target)}, but {SUBJECT($target)} somehow dodge with ludicrous speed!
+petting-failure-holo = You reach out to pet {THE($target)}, but {POSS-ADJ($target) spikes almost impale your hand!
```
Comment (Willhelm53) [inline Resources/Locale/en-US/interaction/interaction-popup-component.ftl:30]: Missing a }

**Review by Willhelm53 — CHANGES_REQUESTED** [review]

**Comment by CrudeWax** [comment]
Now this is what my PR has been lacking all along

**Comment by metalgearsloth** [comment]
changelog text won't show up ingame if that's what you want, needs to follow the formatting in the PR template.

**Pull request merged**